pages/common/ogr2ogr.md

```diff
@@ -0,0 +1,24 @@
+# ogr2ogr
+
+> Converts simple features data between file formats.
+> More information: <https://gdal.org/programs/ogr2ogr.html#ogr2ogr>.
+
+- Convert a Shapefile into a GeoPackage:
+
+`ogr2ogr -f GPKG output.gpkg input.shp`
+
+- Reproject a GeoPackage:
+
+`ogr2ogr -s_srs EPSG:4326 -t_srs EPSG:3857 -f GPKG output.gpkg input.gpkg`
+
+- Convert a CSV file into a GeoPackage and assign projection:
+
+`ogr2ogr -f GPKG output.gpkg input.csv -oo X_POSSIBLE_NAMES=lon -oo Y_POSSIBLE_NAMES=lat -a_srs 'EPSG:4326'`
```

Comment:
```suggestion
`ogr2ogr -f GPKG {{output}}.gpkg {{input.csv}} -oo {{X_POSSIBLE_NAMES=longitude}} -oo {{Y_POSSIBLE_NAMES=latitude}} -a_srs {{'EPSG:4326'}}`
```

Add user token delimiters. Also rename `lon` to `longitude` and `lat` to `latitude` for clarity.